src/norm/pragmas.nim

```diff
@@ -30,8 +30,9 @@ template dbCol*(val: string) {.pragma.}
 template dbType*(val: string) {.pragma.}
   ## DB native type to use in table schema.
 
-template default*(val: string) {.pragma.}
+template default*(val: string) {.pragma, deprecated.}
   ## Default value for the DB column.
+  ## Deprecated since 1.0.18
```

Comment:
Please keep the original docstring and add a notice to it instead of replacing it altogether. Also, please omit hardcoded version numbers since I'm not sure the next version will be 1.0.18 and not 1.1.0.

I suggest this:
```
##[ Default value for the DB column.

**Deprecated.** Default values are set automatically for all columns, this pragma is ignored.
]##
```